samples/config.properties

```diff
@@ -0,0 +1,4 @@
+address=https://grpc.kodypay.com
```

Comment:
Can the default address be `grpc-staging.kodypay.com` ?

Reply:
It can, but I'm not sure we want that. As @kody-joao said, we would like external developers to test their integration in live stores, with real money.